js/src/views/Application/Container/container.js

```diff
@@ -33,7 +33,10 @@ export default class Container extends Component {
     const { children, onCloseFirstRun, showFirstRun, upgradeStore } = this.props;
 
     return (
-      <ParityBackground className={ styles.container }>
+      <ParityBackground
+        attachDocument
```

Comment:
I'm not sure we want this (at least not for every single dapp). Some dapp devs might built their app thinking that the background would be white. There is actually one example of this on Ropsten, which is just some text, black. If the Parity Background is black/dark, the dapp won't render correctly.